src/lib/models/outoforder/Core.cc

```diff
@@ -60,17 +60,23 @@ Core::Core(MemoryInterface& instructionMemory, MemoryInterface& dataMemory,
             dispatchIssueUnit_.forwardOperands(regs, values);
           },
           config["L1-Cache"]["Exclusive"].as<bool>(),
-          config["L1-Cache"]["Load-Bandwidth"].as<uint16_t>(),
-          config["L1-Cache"]["Store-Bandwidth"].as<uint16_t>(),
-          config["L1-Cache"]["Permitted-Requests-Per-Cycle"].as<uint16_t>(),
-          config["L1-Cache"]["Permitted-Loads-Per-Cycle"].as<uint16_t>(),
-          config["L1-Cache"]["Permitted-Stores-Per-Cycle"].as<uint16_t>()),
-      reorderBuffer_(config["Queue-Sizes"]["ROB"].as<unsigned int>(),
-                     registerAliasTable_, loadStoreQueue_,
-                     [this](auto instruction) { raiseException(instruction); }),
+          config["L1-Cache"]["Load-Bandwidth"].as<uint8_t>(),
+          config["L1-Cache"]["Store-Bandwidth"].as<uint8_t>(),
+          config["L1-Cache"]["Permitted-Requests-Per-Cycle"].as<uint8_t>(),
+          config["L1-Cache"]["Permitted-Loads-Per-Cycle"].as<uint8_t>(),
+          config["L1-Cache"]["Permitted-Stores-Per-Cycle"].as<uint8_t>()),
       fetchUnit_(fetchToDecodeBuffer_, instructionMemory, processMemorySize,
-                 entryPoint, config["Core"]["Fetch-Block-Size"].as<uint16_t>(),
+                 entryPoint, config["Fetch"]["Fetch-Block-Size"].as<uint8_t>(),
```

Comment:
Weren't all the config file options updated to take in `uint16_t` rather than `uint8_t`?

Reply:
Good spot, looks like a mistake in rebasing/merging older work